data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -655,6 +655,30 @@
       "actionid": 105,
       "actioncode": "",
       "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUKTA_ADMIN",
+      "actionid": 92,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+      {
+      "rolecode": "MUKTA_ADMIN",
+      "actionid": 93,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUKTA_ADMIN",
+      "actionid": 94,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
```

Comment:
why 93 is required for org admin @divya-egov 
please test and add only required